### PR TITLE
[hi] Add Hindi glossary entry for "manifest"

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
[hi] Add Hindi glossary entry for "manifest"

 
### Description

#Fixes  Localizes content/en/docs/reference/glossary/manifest.md in Hindi content/hi/docs/reference/glossary/manifest.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/manifest.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: मैनिफेस्ट (Manifest)
+id: manifest
+date: 2019-06-28
+short_description: >
+   एक या एक से अधिक क्यूबर्नेट्स एपीआई ऑब्जेक्ट का एक अनुक्रमित विनिर्देश।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/manifest.md:6]:
```suggestion
   एक या एक से अधिक Kubernetes API ऑब्जेक्ट का अनुक्रमित विवरण।
```

File: content/hi/docs/reference/glossary/manifest.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+aka:
+tags:
+- fundamental
+---
+क्यूबर्नेट्स एपीआई ऑब्जेक्ट का विनिर्देश [JSON](https://www.json.org/json-en.html) या [YAML](https://yaml.org/) फ़ॉर्मेट में।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/manifest.md:12]:
```suggestion
Kubernetes API का विवरण [JSON](https://www.json.org/json-en.html) या [YAML](https://yaml.org/) प्रारूप में।
```

File: content/hi/docs/reference/glossary/manifest.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+<!--more-->
+मैनिफेस्ट उस ऑब्जेक्ट की इच्छित स्थिति को निर्दिष्ट करता है जिसे क्यूबर्नेट्स, मैनिफेस्ट को लागू करने पर बनाए रखेगा। YAML फ़ॉर्मेट के लिए, प्रत्येक फ़ाइल में कई मैनिफेस्ट शामिल हो सकते हैं।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/manifest.md:15]:
```suggestion
मैनिफेस्ट उस ऑब्जेक्ट की इच्छित स्थिति को निर्दिष्ट करता है जिसे Kubernetes, मैनिफेस्ट को लागू करने पर बनाए रखेगा। YAML फ़ॉर्मेट के लिए, प्रत्येक कॉन्फ़िगरेशन फ़ाइल में कई मैनिफेस्ट शामिल हो सकते हैं।
```

**Review by niranjandarshann — COMMENTED** [review]
Some suggestion for changes

File: content/hi/docs/reference/glossary/manifest.md (hunk 3 above)
Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/manifest.md]: @niranjandarshann Thank you!! update the changes.

File: content/hi/docs/reference/glossary/manifest.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+   एक या एक से अधिक Kubernetes API ऑब्जेक्ट का अनुक्रमित विवरण।
+
+aka:
+tags:
+- fundamental
+---
+Kubernetes API का विवरण [JSON](https://www.json.org/json-en.html) या [YAML](https://yaml.org/) प्रारूप में।
+
+<!--more-->
+मैनिफेस्ट उस ऑब्जेक्ट की इच्छित स्थिति को निर्दिष्ट करता है जिसे Kubernetes, मैनिफेस्ट को लागू करने पर बनाए रखेगा। YAML फ़ॉर्मेट के लिए, प्रत्येक कॉन्फ़िगरेशन फ़ाइल में कई मैनिफेस्ट शामिल हो सकते हैं।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/manifest.md:15]:
```suggestion
मैनिफेस्ट उस ऑब्जेक्ट की चाही गई स्थिति को निर्धारित करता है जिसे Kubernetes, मैनिफेस्ट को लागू करने पर बनाए रखेगा। YAML फ़ॉर्मेट के लिए, प्रत्येक कॉन्फ़िगरेशन फ़ाइल में कई मैनिफेस्ट शामिल हो सकते हैं।
```

**Review by niranjandarshann — COMMENTED** [review]
one more small suggestion for changes

File: content/hi/docs/reference/glossary/manifest.md (hunk 4 above)
Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/manifest.md]: Done! thanks!

**Review by niranjandarshann — COMMENTED** [review]
Now Changes looks Good To me 
As It uses simple Word and easy to understand instead of typical hindi words.

**Comment by niranjandarshann** [comment]
Thanks For the PR 
/lgtm

**Pull request closed without merging**